mavproxy/source/docs/modules/dataflash_logger.rst

```diff
@@ -8,7 +8,7 @@ DataFlash Logs
     
 This module allows dataflash logs to be downloaded from the autopilot.
 
-The logs will be downloaded to the vehicle logging folder.
+The logs will be downloaded to the vehicle logging folder. In most cases, this has been replaced by the :ref:`FTP <mavproxy-modules-ftp>` module

```

Comment:
I thought ftp log download is much slower....

Reply:
My mistake with the description ... this module is used for livestreaming of the dataflash logs, not downloading logs after the fact.